devchat.go

```diff
@@ -252,6 +258,18 @@ func (u *user) close(msg string) {
 		u.session.Close()
 	})
 }
+func (u *user) system(message string) {
+	u.term.Write([]byte(red.Paint("[SYSTEM] ") + mdRender(message, 0, u.win.Width) + "\n"))
```

Comment:
Why do we _need_ another bot? Let devbot do this too

Reply:
devbot generally looks like a community bot. I intend to leave devbots public announcements and thats what devbot does.

Reply:
devbot _is_ system. devbot is like how slack has slackbot, fun stuff + actual work

besides, I don't want devbot to print the help message, the help message should be printed out with no sender at all

Reply:
Don't resolve until we both agree! What's your opinion?

Reply:
yeah I guess you have a point there. although I personally like [SYSTEM] as the no-sender user.

Reply:
do we even need devbot or system for the output of ./help? Let's just make it look like " -> " for the private messages.